3581/2/commands/command_start.rb

```diff
@@ -0,0 +1,31 @@
+require './verification.rb'
+
+module CommandStart
+  include Verification
+  include Verification::RegistrationProcess
+
+  def start!(*)
+    if register_is_not_required
+      respond_with :message, text: t(:already_registred)
+
+    else
+      save_context :number_input
+      respond_with :message, text: t(:not_registred)
+    end
+  end
+
+  def number_input(number)
+    if correct_number(number)
```

Comment:
Try to think about method names in way you speak. And if method return boolean, add `?` to the method name. 
`if number_is_correct?(number)` sounds better